history.js

```diff
@@ -240,7 +240,7 @@
          */
         "state": {
             get: function() {
-                return stateStorage[windowLocation.href] || null;
+                return typeof stateStorage[windowLocation.href] === 'object' ? JSON.parse(JSON.stringify(stateStorage[windowLocation.href])) : (stateStorage[windowLocation.href] || null);
```

Comment:
Hi.
Adding same changes in `history.ielte7.js` can't be made in the same way as in `history.js` as IE7- doesn't support JSON object. So some cloning function needs to be written. Do you have some preferences in way of cloning objects or you don't care?